<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Paivakirja</class>
 <widget class="QWidget" name="Paivakirja">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Alkupäivä</string>
       </property>
       <property name="buddy">
        <cstring>alkupvm</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="KpDateEdit" name="alkupvm">
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Loppupäivä</string>
       </property>
       <property name="buddy">
        <cstring>loppupvm</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="KpDateEdit" name="loppupvm">
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="kohdennusCheck">
       <property name="text">
        <string>Kohdennuksella</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/kohdennus.png</normaloff>:/pic/kohdennus.png</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="KohdennusCombo" name="kohdennusCombo">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="tiliBox">
       <property name="text">
        <string>Ote tililtä</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/tekstisivu.png</normaloff>:/pic/tekstisivu.png</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="tiliCombo">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Kieli</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="KieliCombo" name="kieliCombo"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="laatuLabel">
       <property name="text">
        <string>Kuvanlaatu</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="LaatuSlider" name="laatuSlider" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="poimiGroup">
     <property name="title">
      <string>Poimi</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>Kaikki</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="tulotRadio">
        <property name="text">
         <string>Tulot</string>
        </property>
        <property name="icon">
         <iconset resource="../pic/pic.qrc">
          <normaloff>:/pic/lisaa.png</normaloff>:/pic/lisaa.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="menotRadio">
        <property name="text">
         <string>Menot</string>
        </property>
        <property name="icon">
         <iconset resource="../pic/pic.qrc">
          <normaloff>:/pic/poista.png</normaloff>:/pic/poista.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="jarjestysRyhma">
     <property name="title">
      <string>Tulostusjärjestys</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="aikajarjestysRadio">
        <property name="text">
         <string>Päivämäärä</string>
        </property>
        <property name="icon">
         <iconset resource="../pic/pic.qrc">
          <normaloff>:/pic/kalenteri16.png</normaloff>:/pic/kalenteri16.png</iconset>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="tositejarjestysRadio">
        <property name="text">
         <string>Tositenumero</string>
        </property>
        <property name="icon">
         <iconset resource="../pic/pic.qrc">
          <normaloff>:/pic/risuaita.png</normaloff>:/pic/risuaita.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="kumppaniCheck">
     <property name="text">
      <string>Tulosta asiakas/toimittaja</string>
     </property>
     <property name="icon">
      <iconset resource="../pic/pic.qrc">
       <normaloff>:/pic/mies.png</normaloff>:/pic/mies.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="tulostakohdennuksetCheck">
     <property name="text">
      <string>Tulosta kohdennukset</string>
     </property>
     <property name="icon">
      <iconset resource="../pic/pic.qrc">
       <normaloff>:/pic/kohdennus.png</normaloff>:/pic/kohdennus.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="alvCheck">
     <property name="text">
      <string>Tulosta alv-prosentti</string>
     </property>
     <property name="icon">
      <iconset resource="../pic/pic.qrc">
       <normaloff>:/pic/vero.png</normaloff>:/pic/vero.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="ryhmittelelajeittainCheck">
     <property name="text">
      <string>Ryhmittele tositelajeittain</string>
     </property>
     <property name="icon">
      <iconset resource="../pic/pic.qrc">
       <normaloff>:/pic/arkisto64.png</normaloff>:/pic/arkisto64.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="tulostasummat">
     <property name="text">
      <string>Tulosta summarivit</string>
     </property>
     <property name="icon">
      <iconset resource="../pic/pic.qrc">
       <normaloff>:/pic/sum.png</normaloff>:/pic/sum.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="eriPaivatCheck">
     <property name="text">
      <string>Erittele päivät tyhjällä rivillä</string>
     </property>
     <property name="icon">
      <iconset resource="../pic/pic.qrc">
       <normaloff>:/pic/Paivakirja64.png</normaloff>:/pic/Paivakirja64.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="piilotaNollaCheck">
     <property name="text">
      <string>Älä näytä nollasaldoja</string>
     </property>
     <property name="icon">
      <iconset resource="../pic/pic.qrc">
       <normaloff>:/pic/sulje.png</normaloff>:/pic/sulje.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="erittelemattomatCheck">
     <property name="text">
      <string>Näytä erittelemättömien viennit</string>
     </property>
     <property name="icon">
      <iconset resource="../pic/pic.qrc">
       <normaloff>:/pic/harmaahuomio.png</normaloff>:/pic/harmaahuomio.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="selvittelyNappi">
       <property name="text">
        <string>Tase-erien selvittely</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/etsi.png</normaloff>:/pic/etsi.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KieliCombo</class>
   <extends>QComboBox</extends>
   <header location="global">kieli/kielicombo.h</header>
  </customwidget>
  <customwidget>
   <class>LaatuSlider</class>
   <extends>QWidget</extends>
   <header location="global">arkistoija/laatuslider.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KohdennusCombo</class>
   <extends>QComboBox</extends>
   <header location="global">tools/kohdennuscombo.h</header>
  </customwidget>
  <customwidget>
   <class>KpDateEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">tools/kpdateedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>alkupvm</tabstop>
  <tabstop>loppupvm</tabstop>
  <tabstop>kohdennusCheck</tabstop>
  <tabstop>kohdennusCombo</tabstop>
  <tabstop>tiliBox</tabstop>
  <tabstop>tiliCombo</tabstop>
  <tabstop>kieliCombo</tabstop>
  <tabstop>radioButton</tabstop>
  <tabstop>tulotRadio</tabstop>
  <tabstop>menotRadio</tabstop>
  <tabstop>aikajarjestysRadio</tabstop>
  <tabstop>tositejarjestysRadio</tabstop>
  <tabstop>kumppaniCheck</tabstop>
  <tabstop>tulostakohdennuksetCheck</tabstop>
  <tabstop>alvCheck</tabstop>
  <tabstop>ryhmittelelajeittainCheck</tabstop>
  <tabstop>tulostasummat</tabstop>
  <tabstop>eriPaivatCheck</tabstop>
  <tabstop>piilotaNollaCheck</tabstop>
  <tabstop>selvittelyNappi</tabstop>
 </tabstops>
 <resources>
  <include location="../pic/pic.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>kohdennusCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>kohdennusCombo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>561</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tiliBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>tiliCombo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
